test/integration/pubsub_scheduled/inspec.yml

```
@@ -2,11 +2,11 @@ name: pubsub_scheduled
 depends:
   - name: inspec-gcp
     git: https://github.com/inspec/inspec-gcp.git
-    version: ~> 0.9.0
+    version: ~> 0.11.0
```

`version` is not actually supported by the git dependency, contrary to InSpec documentation. Additionally, the [v0.11.0 release is currently broken](https://github.com/inspec/inspec-gcp/issues/160).

```suggestion
    tag: v0.10.0
```

---

This has worked for me

---

This will not error; `version` is simply ignored by InSpec. As this change currently has no functional impact, I prefer that we correct the configuration.

---

I has impact: it FIXED issue for me